implementation/scripts/package.sh

```diff
@@ -64,20 +64,25 @@ function main {
 
   tools::install "${token}"
 
-  buildpack::archive "${version}"
-  buildpackage::create "${output}"
+  buildpack_type=buildpack
+  if [ -f "extension.toml" ]; then
```

Comment:
Does this need to be an absolute path as well to solve the same problem as seen in https://github.com/paketo-buildpacks/github-config/pull/947

Reply:
@ForestEckhardt There was no error on that one, but I changed it anyway to point to an absolute path. I also rebased the branch.